### PR TITLE
Add adblocker and 0/single node disclaimers

### PR DESCRIPTION
**Pull request by everett980**
## Which problem is this PR solving?
- DDG request is blocked by some adblockers
- Plexus does not render single vertex

## Short description of the changes
- Add disclaimer to whitelist Jaeger on DDG request failure details view.
image
("New!" banner not in diff)
- Add disclaimer when there are zero or fewer vertices to view.
No dependencies found:
image
Dependencies found but hidden:
image

This PR is branched off of [another PR](https://github.com/jaegertracing/jaeger-ui/pull/501), [this commit contains the diff for this PR](https://github.com/jaegertracing/jaeger-ui/pull/502/commits/00a51f62bbd88e175edca6fa5285223f3de6112c)

### CODE REVIEW

**Comment by yurishkuro** [comment]
there seems to be many changes not related to just disclaimers

**Comment by everett980** [comment]
@yurishkuro the PR is branched off of another PR. There is a link in the PR description to the commit that contains the disclaimer changes.

**Comment by everett980** [comment]
The middle branch was merged, so the diff is good now (except for two files with TODO delete changes comments that I definitely deleted on the middle branch.....)

File: packages/jaeger-ui/src/constants/default-config.tsx [hunk 1]
```diff
@@ -20,6 +20,10 @@ export default deepFreeze(
   Object.defineProperty(
     {
       archiveEnabled: false,
+      // TODO: remove default
+      deepDependencies: {
+        menuEnabled: true,
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/constants/default-config.tsx]: Do we really want that in oss?  There's no backend to generate DDGs.

Comment (everett980) [reply packages/jaeger-ui/src/constants/default-config.tsx]: yeah hence the TODO and comment above that I thought TODOs were dealt with...
definitely won't have this on by default. same with the `New!` css

Comment (everett980) [reply packages/jaeger-ui/src/constants/default-config.tsx]: Fixed

**Pull request merged**